runtime/src/lib.rs

```diff
@@ -164,12 +205,16 @@ impl_opaque_keys! {
     }
 }
 
+/// Implementations of some helper traits passed into runtime modules as associated types.
+pub mod impls;
+pub use impls::Author;
+
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("datahighway-parachain"),
     impl_name: create_runtime_str!("datahighway-parachain"),
-    authoring_version: 2,
-    spec_version: 2,
+    authoring_version: 3,
```

Comment:
Since we will be doing a fresh release, shouldn't these all values be 1

Reply:
yes we could could change it back to defaults https://github.com/substrate-developer-hub/substrate-parachain-template/blob/main/runtime/src/lib.rs#L172

Reply:
please change it back to defaults

Reply:
its reset to defaults